src/core_plugins/metrics/public/components/vis_editor.js

```diff
@@ -47,7 +47,7 @@ class VisEditor extends Component {
     this.onBrush = brushHandler(props.vis.API.timeFilter);
     this.handleUiState = this.handleUiState.bind(this, props.vis);
     this.handleAppStateChange = this.handleAppStateChange.bind(this);
-    this.getConfig = (...args) => props.config.get(...args);
+    this.getConfig = (...args) => this.props.config.get(...args);
```

Comment:
What if we move this as a function of vis_editor, instead of creating it into the constructor.
We can also add some checks on the `config` object to check if it exist before using it.

Reply:
np, I just wanted to minimize the diff. About adding the additional check for the 'config' obj not sure that it's a good idea. I can add this check but I cannot understand what you want to return in case of undefined config. For this case probably better to see exception in the console to understand why config is not defined...